refactor and upgrade honeycomb-console version

ideal opened the pull request:
升级honeycomb-console的版本，处理server_ctl的几个问题。

ideal committed:
refactor and upgrade honeycomb-console version

cctv1005s commented:
LGTM

cctv1005s commented on honeycomb-server/Makefile:
```diff
@@ -1,2 +1,5 @@
\ No newline at end of file
+	@docker build . -t node2honeycomb/honeycomb-server:latest
+
+buildx:
```
Should be `build-amd:`

ideal replied:
嗯主要因为honeycomb-server.rpm这个包只有x86的，没有arm64的。honeycomb-server.rpm这个包你在哪构建的？

cctv1005s replied:
github action 上跑的还是内网的机器，我给忘了

ideal committed:
add arm64 centos repos, but rpm install still fail

0 0.111 error: Failed dependencies:
0 0.111        libc.so.6(GLIBC_2.10)(64bit) is needed by honeycomb-server-1.2.4_1-2.el7.x86_64
0 0.111        libc.so.6(GLIBC_2.14)(64bit) is needed by honeycomb-server-1.2.4_1-2.el7.x86_64
0 0.111        libc.so.6(GLIBC_2.2.5)(64bit) is needed by honeycomb-server-1.2.4_1-2.el7.x86_64
0 0.111        libc.so.6(GLIBC_2.3)(64bit) is needed by honeycomb-server-1.2.4_1-2.el7.x86_64
0 0.111        libc.so.6(GLIBC_2.3.2)(64bit) is needed by honeycomb-server-1.2.4_1-2.el7.x86_64
0 0.111        libc.so.6(GLIBC_2.3.4)(64bit) is needed by honeycomb-server-1.2.4_1-2.el7.x86_64
0 0.111        libc.so.6(GLIBC_2.4)(64bit) is needed by honeycomb-server-1.2.4_1-2.el7.x86_64
0 0.111        libc.so.6(GLIBC_2.7)(64bit) is needed by honeycomb-server-1.2.4_1-2.el7.x86_64
0 0.111        libcrypt.so.1(GLIBC_2.2.5)(64bit) is needed by honeycomb-server-1.2.4_1-2.el7.x86_64
0 0.111        libdl.so.2(GLIBC_2.2.5)(64bit) is needed by honeycomb-server-1.2.4_1-2.el7.x86_64
0 0.111        libpthread.so.0(GLIBC_2.2.5)(64bit) is needed by honeycomb-server-1.2.4_1-2.el7.x86_64
